src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/prisonperson/physicalattributes/PhysicalAttributesDpsApiService.kt

```diff
@@ -4,14 +4,14 @@ import org.springframework.beans.factory.annotation.Qualifier
 import org.springframework.stereotype.Service
 import org.springframework.web.reactive.function.client.WebClient
 import uk.gov.justice.digital.hmpps.prisonertonomisupdate.helpers.awaitBodyOrNullForNotFound
-import uk.gov.justice.digital.hmpps.prisonertonomisupdate.prisonperson.model.PhysicalAttributesDto
+import uk.gov.justice.digital.hmpps.prisonertonomisupdate.prisonperson.model.PhysicalAttributesSyncDto
 
 @Service
 class PhysicalAttributesDpsApiService(@Qualifier("prisonPersonApiWebClient") private val webClient: WebClient) {
-  suspend fun getPhysicalAttributes(prisonerNumber: String): PhysicalAttributesDto? =
+  suspend fun getPhysicalAttributes(prisonerNumber: String): PhysicalAttributesSyncDto? =
     webClient
       .get()
-      .uri("/prisoners/{prisonerNumber}/physical-attributes", prisonerNumber)
+      .uri("/sync/prisoners/{prisonerNumber}/physical-attributes", prisonerNumber)
       .retrieve()
-      .awaitBodyOrNullForNotFound<PhysicalAttributesDto>()
+      .awaitBodyOrNullForNotFound<PhysicalAttributesSyncDto>()
```

Comment:
We now have a dedicated endpoint for getting sync and reconciliation data from the DPS API.